wurst/event/DamageType.wurst

```diff
@@ -138,7 +139,7 @@ function handler() returns bool
 
         let sampledLife = tU.getHP() - .405
         if sampledLife >= attemptedDamage and sampledLife <= 2.*attemptedDamage
-            tU.setHP(sampledLife - attemptedDamage)
+            tU.setHP(sampledLife - attemptedDamage + .405)
```

Comment:
Why did you change this?

Reply:
I noticed units lose a bit more health from any damage than they should.
Why would you ever want to remove that .405 points of life from unit?
I didn't test that, but without my fix a 0.01 damage should kill unit with 0.5 hp left